src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
```

Comment:
Don't apply any styles except box-sizing to the * selector, you can apply font-family and font-weight to the html tag

Reply:
I made changes but now I fail github checks. Although npm test works fine on my computer, github tells me that all checks have failed. Can You explain me why?